src/app/not-found/not-found.component.scss

```diff
@@ -0,0 +1,14 @@
+.center {
+  position: absolute;
```

Comment:
Try to find a simpler way to center the text. Mentioning so many properties might not be necessary.